agency/README.md

```diff
@@ -231,17 +232,47 @@ A standard point of vehicle telemetry. References to latitude and longitude impl
 | `gps.hdop`     | Float          | Required if Available | Horizontal GPS or GNSS accuracy value (see [hdop][hdop]) |
 | `gps.satellites` | Integer      | Required if Available | Number of GPS or GNSS satellites
 | `charge`       | Float          | Required if Applicable | Percent battery charge of vehicle, expressed between 0 and 1 |
+| `stop_id`      | UUID           | Required if Applicable | Stop that the vehicle is currently located at. Only applicable for _docked_ Micromobility. See [Stops][stops] |
+
+## Stops
+
+The `/stops` endpoint allows an agency to register Stops.
```

Comment:
@avatarneil I think we forgot the wording update here, could you add a quick patch with it?

```suggestion
The `/stops` endpoint allows an agency to register city-managed Stops, or a provider to register self-managed Stops.
```

Reply:
I've made this update in the release cleanup PR here: https://github.com/openmobilityfoundation/mobility-data-specification/pull/539/commits/983a99fbb84d6d2ea805bcb82676a63bb431a605